.gitignore

```diff
@@ -48,6 +48,8 @@ debug.log
 npm-debug.log
 .npmrc*
 .env*
+engflow.crt
```

Comment:
Can we do *.crt and *.key?

Reply:
We don't seem to have any other CRT or KEY files in the repo so let's add with wildcard.

Reply:
Good idea. I've kept `engflow.crt` as a non-wildcard since it would be okay to commit public keys, but private keys are problematic (outside of tests).

Reply:
I also added `privkey.pem` by itself. There are some private keys in the codebase but not with that exact name.